src/common/gdicmn.cpp

```diff
@@ -328,6 +328,7 @@ void wxColourDatabase::Initialize()
         {wxT("INDIAN RED"), 79, 47, 47},
         {wxT("KHAKI"), 159, 159, 95},
         {wxT("LIGHT BLUE"), 191, 216, 216},
+        {wxT("LIGHT GREEN"), 0, 128, 0},
```

Comment:
We could still apply this (and the other changes in this file) as a separate commit but IMO they're not necessary for this PR, see below.